README.md

```diff
@@ -1,6 +1,33 @@
 # TOVE: Transcription Object Viewer/Editor
 
-*Readme and docs coming soon*	
+Tove is the API for the [ALICE Text Editor](https://github.com/zooniverse/text-editor) tool. The text editor tool takes in reductions on transcription data from Caesar, and allows users to review, update and approve transcriptions.
+
+## Setup
+
+### Run app locally
+
+1. Install dependencies:
+- Rails 6.0.1
+- Ruby 2.6.5
+- PostgreSQL 9.5
+
+2. Clone the repo `git clone https://github.com/zooniverse/tove.git` 
+
+3. `cd` into the cloned folder   
+
+4. Run `bundle install`  
+
+5. Run the following commands to set up the database and generate test data:
+```
+rake db:setup
+rake db:migrate
```

Comment:
`rake db:setup` will create the db, load the schema and then seed the data. https://github.com/rails/rails/blob/51d73fb84bb3f552efe44bb461b36caa9570928e/activerecord/lib/active_record/railties/databases.rake#L297

So no need for the extras, unless you have pending migrations that aren't in the schema  but that's not really getting started.

Reply:
Ahh, I see - that makes sense.